sdk/src/main/java/com/hedera/hashgraph/sdk/Query.java

```diff
@@ -270,6 +270,11 @@ TransactionId getTransactionIdInternal() {
         return paymentTransactionId;
     }
 
+    @Nullable
+    TransactionId getTransactionID() {
```

Comment:
This should be `getTransactoinId` with a lowercase `D`. Also, mark this as `public`

Reply:
OOPS. Habit from my old job.